extensions/libraries/redcore/factory/factory.php

```diff
@@ -102,4 +102,51 @@ protected static function createDbo()
 
 		return $db;
 	}
+
+	/**
+	 * Create a mailer object
+	 *
+	 * @return  JMail object
+	 *
+	 * @see     JMail
+	 * @since   11.1
+	 */
+	protected static function createMailer()
```

Comment:
not really, this should be extending JFactory function with the same name so we can use our RMail with BC in it